### ISSUE TITLE
BulkReadAsync on PostgreSql return error

### ISSUE THREAD

**Issue by dseferlis** [borisdj/EFCore.BulkExtensions#667]
Hello all and thank you for your work on BulkExtensions.
I'm trying to use BulkExtensions with PosgreSQL. I have the following entity class:
XmlProduct.cs
```
public class XmlProduct
{
        public string AtlantisCode { get; set; }  //unique - key
        public string Title { get; set; }
        public string Description { get; set; }
etc...
}
```
I load some data from xml and the insert into db using BulkInsertOrUpdateAsync which works fine. Then I need based on xml AtlantisCodes to bulk update some object properties. So trying to bulk read the db so I can get the objects to update.

```
var readConfig = new BulkConfig
{
    UpdateByProperties = new List<string> { nameof(XmlProduct.AtlantisCode) }
};

var activeItems = xmlAtlantisCodes.Select(x => new XmlProduct { AtlantisCode = x }).ToList(); //xmlAtlantisCodes is a List<string>

await _appContext.BulkReadAsync(activeItems, readConfig);
```
But on BulkReadAsync I get the error:
```
System.InvalidOperationException: The required column 'AtlantisCode' was not present in the results of a 'FromSql' operation.
```
On SqlServer db it's working fine. Am I missing something Postgresql specific or bulkread is not impemented on Postgresql yet?
Thanks in advance.

**Comment by borisdj** [comment]
It should work.
Check if you have the latest version, and try make a reproducible test or alter exiting one:
https://github.com/borisdj/EFCore.BulkExtensions/blob/622f651ba8e0c0cc965538f005a7c966872a5f87/EFCore.BulkExtensions.Tests/EFCoreBulkTest.cs#L113

**Comment by dseferlis** [comment]
Changing Test to this:
```
// READ
var configReadBy = new BulkConfig { UpdateByProperties = new List<string> { nameof(Item.ItemId) } };
var secondEntity = new List<Item>() { new Item(){ ItemId = 2} };
//Assert.Equal(0, secondEntity.FirstOrDefault().ItemId);
context.BulkRead(secondEntity, configReadBy);
Assert.Equal(2, secondEntity.FirstOrDefault().ItemId);
```
It does return the same error.

Also, in unmodified test, noticed that Postgres BulkRead only queries the entity key. On this test the ItemId. No other fields. On SQLite or SQLServer it queries all fields.

**Comment by borisdj** [comment]
Should be now fixed with 6.1.0

**Comment by dseferlis** [comment]
Checked tests again. Postgres only queries primary key. Not all properties. Is this still bug or it's the way it's working?
My change to test was this:
```
// READ
var secondEntity = new List<Item>() { new Item { Name = entities[1
Assert.Equal(0, secondEntity.FirstOrDefault().ItemId);
context.BulkRead(secondEntity, configUpdateBy);
Assert.Equal(2, secondEntity.FirstOrDefault().ItemId);
```
Thanks
